packages/miniflare/test/plugins/email/index.spec.ts

```diff
@@ -263,6 +263,49 @@ This is a random email body.
 	t.is(res.status, 200);
 });
 
+test("Multiple allowed senders send_email binding works", async (t) => {
```

Comment:
We should also have a test for making sure that using a not allowed sender triggers the error

Reply:
Added new test.